src/ucp/api/ucp.h

```diff
@@ -2171,13 +2243,27 @@ void ucp_request_release(void *request);
  * called with the @a status argument of the callback set to UCS_OK, and in a
  * case it is canceled the @a status argument is set to UCS_ERR_CANCELED.  It is
  * important to note that in order to release the request back to the library
- * the application is responsible to call @ref ucp_request_release
- * "ucp_request_release()".
+ * the application is responsible to call @ref ucp_request_free
+ * "ucp_request_free()".
  */
 void ucp_request_cancel(ucp_worker_h worker, void *request);
 
 
 /**
+ * @ingroup UCP_COMM
+ * @brief Release a communications request.
+ *
+ * @param [in]  request      Non-blocking request to release.
+ *
+ * This routine releases the non-blocking request back to the library, regardless
+ * of its current state. Communications which were started by this request will
```

Comment:
Communications which were started by this request -> Communication operations associated with this request